src/Core/src/Platform/Android/ViewExtensions.cs

```diff
@@ -14,6 +15,17 @@ public static void UpdateIsEnabled(this AView nativeView, IView view)
 				nativeView.Enabled = view.IsEnabled;
 		}
 
+		public static void UpdateIsVisible(this AView nativeView, IView view)
+		{
+			if (view.IsVisible && nativeView.Visibility != ViewStates.Visible)
+				nativeView.Visibility = ViewStates.Visible;
+			if (!view.IsVisible && nativeView.Visibility != ViewStates.Gone)
+				nativeView.Visibility = ViewStates.Gone;
+
+			nativeView.Invalidate();
+			nativeView.RequestLayout();
```

Comment:
Is this the correct set of operations?

Also, we have all the checks above, but then still just fire a re-layout. We probably should only do this when it actually changed.